tests/features/support/steps.js

```diff
@@ -20,9 +20,9 @@ defineStep(/^I (?:find|identify|see|spot).*? (`.*`).*?$/, selectorChain =>
   client.expect.element(nestedSelector(selectorChain)).to.be.visible);
 
 defineStep(/^I (?:can|don)'t (?:find|identify|see|spot).*? (`.*`).*?$/, selectorChain =>
-  client.expect.element(nestedSelector(selectorChain)).to.not.be.visible);
+  client.expect.element(nestedSelector(selectorChain)).to.not.be.present);
```

Comment:
Are we sure about this change? The step text is about finding/seeing/etc rather than presence/absence in the HTML.

Reply:
I think we may want both present and visible checks.

However for this situation '.not.be.visible' causes an error because it can't find the element to check visibility for.